src/Qir/Common/cmake/unit_test_include.cmake

```diff
@@ -9,7 +9,7 @@ macro(add_unit_test target)
     if(DEFINED ENV{NATIVE_SIMULATOR})
         set(TEST_DEPS1 $ENV{NATIVE_SIMULATOR})
     else()
-        set(TEST_DEPS1 "${PROJECT_SOURCE_DIR}/../../Simulation/native/build/$ENV{BUILD_CONFIGURATION}")
+        set(TEST_DEPS1 "${PROJECT_SOURCE_DIR}/../../Simulation/native/build/drop")
```

Comment:
@kuzminrobin Why did this path change? Seems like the CI should have had issues before or after.

Reply:
When the native simulator was built with MSVC (on Win), the compiler was putting the binary to
`Simulation/native/build/Release` (whereas the GCC on other platforms was putting the binary to 
`Simulation/native/build`). Then the scripts were copying the binary to 
`Simulation/native/build/drop` (on all platforms), but the dependency in the tests was different for win and other platforms. 
After the native simulator has been migrated to Clang, on all the platforms the binary started being put 
`Simulation/native/build` and copied to 
`Simulation/native/build/drop`. This change has unified the dependency.